kyuubi-common/src/test/scala/org/apache/kyuubi/operation/JDBCTests.scala

```diff
@@ -379,4 +379,31 @@ trait JDBCTests extends BasicJDBCTests {
       assert(rs.getString(1) == System.getProperty("user.name"))
     }
   }
+
+  test("KYUUBI #1059: Plan only operations") {
+    val ddl = "create table t(a int) using parquet"
+    val dql = "select * from t"
+    withJdbcStatement("t") { statement =>
+      try {
+        statement.execute("SET kyuubi.operation.plan.only.mode=optimize")
+        val set = statement.executeQuery(ddl)
+        assert(set.next())
+        assert(set.getString("plan") startsWith "Create")
```

Comment:
Is the result stable? If not stable, can we paste a demo result here?

Reply:
If assert fails, we can see the original plan